e2e/specs/drug-orders.spec.ts

```diff
@@ -222,6 +222,140 @@ test('Record, edit and discontinue a drug order', async ({ page }) => {
+      .getByRole('button', { name: /add to basket/i })
+      .click();
+  });
+
+  await test.step('Then I should see "Aspirin 81mg" under drug order', async () => {
+    await expect(page.getByText('Aspirin 81mg')).toBeVisible();
+  });
+
+  await test.step('When I click on the newly created drug order', async () => {
+    await page
+      .getByRole('listitem')
+      .filter({ hasText: /incomplete/i })
+      .click();
+  });
+
+  await test.step('Then I should see the drug order form launch in the workspace', async () => {
+    await expect(page.getByText(/order form/i)).toBeVisible();
+  });
+
+  await test.step('When I set the dose to "1" tablet', async () => {
+    await page.getByLabel(/^dose$/i, { exact: true }).click();
+    await page.getByLabel(/^dose$/i, { exact: true }).fill('1');
+  });
+
+  await test.step('And I set the route to "Oral"', async () => {
+    await page.getByRole('combobox', { name: /route/i }).click();
+    await page.getByRole('option', { name: /oral/i }).click();
+  });
+
+  await test.step('And I set the frequency to "Once daily"', async () => {
+    await page.getByRole('combobox', { name: /frequency/i }).click();
+    await page.getByRole('option', { name: /once daily/i }).click();
+  });
+
+  await test.step('And I set duration to "3" days', async () => {
+    await page.getByText(/^duration$/i).clear();
+    await page.getByText(/^duration$/i).fill('3');
+  });
+
+  await test.step('And I set the quantity to dispense to 3', async () => {
+    await page.getByText(/^quantity to dispense$/i).clear();
+    await page.getByText(/^quantity to dispense$/i).fill('3');
+  });
+
+  await test.step('And I set the prescription refills to 1', async () => {
+    await page.getByText(/^prescription refills$/i).clear();
+    await page.getByText(/^prescription refills$/i).fill('1');
+  });
+
+  await test.step('And I set the indication to "Headache"', async () => {
+    await page.getByText(/indication/i).fill('Headache');
+  });
+
+  await test.step('And I click on the "Save Order" button', async () => {
+    await page.getByRole('button', { name: /save order/i }).click();
+  });
+
+  await test.step('Then the order status should be changed to "New"', async () => {
+    await expect(orderBasket.getByText(/incomplete/i)).toBeHidden();
+    await expect(orderBasket.getByText(/new/i)).toBeVisible();
+  });
+
+  await test.step('When I click on the "Sign and close" button', async () => {
+    await page.getByRole('button', { name: /sign and close/i }).click();
+  });
+
+  await test.step('Then I should see a success notification', async () => {
+    await expect(page.getByText(/placed order for aspirin/i)).toBeVisible();
+  });
+
+  await test.step('And I should see the newly added order in the active medications list', async () => {
+    const headerRow = medicationsPage.medicationsTable().locator('thead > tr');
+    const dataRow = medicationsPage.medicationsTable().locator('tbody > tr');
```

Comment:
Why are these rules defined using `medicationsPage`. I think we can just use `page` here.